components/homme/src/theta-l/CMakeLists.txt

```diff
@@ -12,54 +12,91 @@ SET(UTILS_TIMING_DIR  ${CMAKE_BINARY_DIR}/utils/cime/src/share/timing)
 # Make INCLUDE_DIRS global so the tests can access it
 SET (EXEC_INCLUDE_DIRS ${PIO_INCLUDE_DIRS} ${UTILS_TIMING_DIR} )
 
-# Find F90 files in target directory
```

Comment:
Here is where the change to avoid all the files in the theta-l directory from being compiled (to hopefully prevent the ARKode interface files from being compiled if HOMME_USE_ARKODE is set to false)